src/views/Pools/Syrup.tsx

```diff
@@ -76,7 +76,7 @@ const Farm: React.FC = () => {
             <li>{TranslateString(406, 'Rewards are calculated per block.')}</li>
           </ul>
         </div>
-        <img src="/images/syrup.png" alt="SYRUP POOL icon" />
+        <img src="/images/syrup.png" alt="SYRUP POOL icon" width={410} height={191} />
```

Comment:
Will this scale? Does it need to?

Reply:
it will: https://github.com/pancakeswap/pancake-frontend/blob/develop/src/style/Global.tsx#L17